src/main/java/com/questhelper/helpers/quests/deathontheisle/DeathOnTheIsle.java

```diff
@@ -0,0 +1,734 @@
+		talkToCostumerAgain.addSubSteps(talkToCostumerAboutStainedCostume);
+		var talkedAboutStainedCostume = new VarbitRequirement(11256, 1);
+
+		var talkToCostumerAboutHiddenPassage = talkToCostumerAgain.copy();
+		talkToCostumerAboutHiddenPassage.addDialogStep("Did you know there was a hidden passage in here?");
+		talkToCostumerAgain.addSubSteps(talkToCostumerAboutHiddenPassage);
+		// var talkedAboutHiddenPassage = new VarbitRequirement(11254, 1);
+
+		investigateTheatreCellar = new ConditionalStep(this, talkToCostumerAboutHiddenPassage);
+		investigateTheatreCellar.addStep(not(inVilla), returnToButlerAndHeadInside);
+		investigateTheatreCellar.addStep(aroundVilla, climbFirstLooseRocksToTheatre);
+		investigateTheatreCellar.addStep(outsideVillaByLooseRocks, climbSecondLooseRocksToTheatre);
+		investigateTheatreCellar.addStep(not(inTheatreCellar), enterBackstage);
+		investigateTheatreCellar.addStep(not(searchedCrateNextToStairs), searchCrateNextToStairs);
+		investigateTheatreCellar.addStep(not(searchedBookshelf), searchBookshelf);
+		investigateTheatreCellar.addStep(not(searchedCostumeRack), searchCostumeRack);
+		investigateTheatreCellar.addStep(not(talkedAboutActors), talkToCostumerAboutActors);
+		investigateTheatreCellar.addStep(not(talkedAboutPoison), talkToCostumerAboutPoison);
+		investigateTheatreCellar.addStep(not(talkedAboutStainedCostume), talkToCostumerAboutStainedCostume);
+
+		/// 40
+		var climbUpFromTheatreCellar = new ObjectStep(this, ObjectID.STAIRS_54717, new WorldPoint(1469, 9328, 0), "Climb up the stairs of the theatre cellar.");
+
+		// TODO: can we only highlight "More options" if "Naiatli" is not one of the options?
+		speakToGuards = new NpcStep(this, NpcID.STRADIUS, new WorldPoint(1472, 2925, 0), "Report back to Stradius near the theatre and accuse Naiatli.");
+		speakToGuards.addDialogStep("More options...");
+		speakToGuards.addDialogStep("Naiatli.");
+		speakToGuards.addSubSteps(climbUpFromTheatreCellar);
+
+		// varbit 11257=1 if spoken to stradius a bit
+
+		snitchToGuardsStep = new ConditionalStep(this, speakToGuards);
+		snitchToGuardsStep.addStep(not(inVilla), returnToButlerAndHeadInside);
+		snitchToGuardsStep.addStep(aroundVilla, climbFirstLooseRocksToTheatre);
+		snitchToGuardsStep.addStep(outsideVillaByLooseRocks, climbSecondLooseRocksToTheatre);
+		snitchToGuardsStep.addStep(inTheatreCellar, climbUpFromTheatreCellar);
+
+		// 42 + 45
+		talkToStradiusToEnterTheTheatre = new NpcStep(this, NpcID.STRADIUS, new WorldPoint(1472, 2925, 0), "Talk to Stradius to enter the theatre.");
+		confrontNaiatli = new NpcStep(this, NpcID.NAIATLI, new WorldPoint(1465, 2932, 0), "Confront Naiatli, attacking her when she runs away. When Clodius appears, attack him instead. Use Protect from Melee to avoid all damage.");
+		var killClodius = new NpcStep(this, NpcID.CLODIUS, new WorldPoint(1469, 2937, 0), "Kill Clodius. Use Protect from Melee to avoid all damage.");
```

Comment:
You just ever attack him, so I think this should be 'Attack Clodius.'

Reply:
Changed phrasing to "Attack" instead of "Kill" in [`5e7e68a` (#1774)](https://github.com/Zoinkwiz/quest-helper/pull/1774/commits/5e7e68a2bccbce7f52739851eb498ada919e4e5e)